cli/user42_subdomain_template.txt

```diff
@@ -0,0 +1,6 @@
+http://<subdomain>.<nest_user>.hackclub.dn42 {
+	bind unix/.<subdomain>.webserver.sock|777
```

Comment:
Would this cause issues if a user already has a non-dn42 subdomain with the same name?

Reply:
I don't think so, since they're different commands (add vs add42)

Reply:
The webserver socket names would conflict, though

Reply:
Oh I see what you're saying. Will fix, thanks.

Reply:
I think this should be optional behavior - you might want both to be on the same socket or seperate but by default this is incorrect behavior